hw/top_earlgrey/data/ip/chip_flash_ctrl_testplan.hjson

```diff
@@ -118,6 +118,7 @@
             - Ensure that the low power entry does not happen due to the ongoing flash operation.
             '''
       stage: V2
+      si_stage: None
```

Comment:
What does "None" mean here? Are these tests that could not be performed in silicon? I'm not familiar with the `si_stage` label.

Reply:
yep, follow @matutem 's suggestion